docs/copilot/workspace-context.md

```diff
@@ -0,0 +1,75 @@
+---
+Order: 7
+Area: copilot
+TOCTitle: Workspace Context
+ContentId: c77dcce9-4ba9-40ac-8ae5-2df855088090
+PageTitle: Chat using @workspace Context References
+DateApproved: 02/28/2024
+MetaDescription: How to use Copilot's @workspace chat to ask questions against your entire codebase.
+---
+# Making Copilot Chat an expert in your workspace
+
+Referencing `@workspace` in Copilot Chat lets you ask questions about your entire codebase. Based on the question Copilot intelligently retrieves relevant files and symbols, which it then references in its answer as links and code examples. Grounded in `@workspace` references, Copilot Chat becomes a domain expert for tasks like:
+
+- Finding existing code in your codebase:
+  - `"@workspace where is database connecting string configured?"` - Explains where and how the database connection is configured
+  - `"@workspace how can I validate a date?"` - Finds existing date validation helpers in the codebase
+  - `"@workspace where are tests defined?"` - Provides the location of test suites, cases, and related references and configurations
+- Making plans for complex code edits:
+  - `"@workspace how can I add a rich tooltip to a button?"` - Provides a plan for using the existing tooltip component with button elements
+  - `"@workspace add date validation to #selection"` - Plans how to apply the existing date validation to the selected code
+  - `"@workspace add a new API route for the forgot password form"` - Outlines where to add the new route and how to connect it to the existing code
+- Explaining higher-level concepts in a codebase:
+  - `"@workspace how is authentication implemented?"` - Overview of the authentication flow and references to the relevant code
+  - `"@workspace which API routes depend on this service?"` - Lists the routes that use the service in the selected code
+  - `"How do I build this #codebase?"` - List the steps to build the project based on documentation, scripts, and configurations
```

Comment:
Sorry if I missed a reply to my previous comment here - this example stands out to me because it doesn't specifically use `@workspace`. Might want to explain that using `#codebase` is an alternative?